client/web/compose/src/components/PageBlocks/RecordBase.vue

```diff
@@ -224,3 +296,25 @@ export default {
   min-height: 1.2rem;
 }
 </style>
+
+<style lang="scss" scoped>
+.field-label{
+  display: flex;
+  gap: 20px;
+  align-items: center;
+}
+
+.inline-actions {
+  width: fit-content;
+  opacity: 0;
+  transition: opacity 0.25s;
+}
+
+.field-label:hover .inline-actions {
```

Comment:
It should be displayed when you hover over the field element, not just the label part)